File: go-client-sdk/pkg/models/shared/projectrunstatusenum.go
package shared

type ProjectRunStatusEnum string

const (
	ProjectRunStatusEnumPending                ProjectRunStatusEnum = "PENDING"
	ProjectRunStatusEnumRunning                ProjectRunStatusEnum = "RUNNING"
	ProjectRunStatusEnumErrored                ProjectRunStatusEnum = "ERRORED"
	ProjectRunStatusEnumCompleted              ProjectRunStatusEnum = "COMPLETED"
	ProjectRunStatusEnumKilled                 ProjectRunStatusEnum = "KILLED"
	ProjectRunStatusEnumUnableToAllocateKernel ProjectRunStatusEnum = "UNABLE_TO_ALLOCATE_KERNEL"
)

File: typescript-client-sdk/src/sdk/models/shared/projectrunstatusenum.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";

export enum ProjectRunStatusEnum {
    Pending = "PENDING",
    Running = "RUNNING",
    Errored = "ERRORED",
    Completed = "COMPLETED",
    Killed = "KILLED",
    UnableToAllocateKernel = "UNABLE_TO_ALLOCATE_KERNEL"
}